<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clavier</class>
 <widget class="QWidget" name="Clavier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Centrale d'Alarme</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>256</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox_LedVerte">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator{background-color: #00FF00;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Code">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {font-style: italic;color:#8f8f8f};</string>
        </property>
        <property name="text">
         <string>Code : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_LedRouge">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox::indicator{background-color: #FF0000;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Valide">
      <property name="text">
       <string>Entrer Code</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {border: 2px solid #000000; }</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {border: 2px solid #000000; }</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_6">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {border: 2px solid #000000; }</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_1">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {border: 2px solid #000000; }</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {border: 2px solid #000000; }</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_9">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {border: 2px solid #000000; }</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_7">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {border: 2px solid #000000; }</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_8">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {border: 2px solid #000000; }</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_5">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {border: 2px solid #000000; }</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_Marche">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {border: 3px solid #00FF00; }</string>
        </property>
        <property name="text">
         <string>Marche</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_0">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {border: 2px solid #000000; }</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_Arret">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed {border: 3px solid #FF0000; }</string>
        </property>
        <property name="text">
         <string>ArrÃªt</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
